xml/security_kerberos.xml

```diff
@@ -1914,6 +1914,140 @@ Valid starting     Expires            Service principal
+       </step>
+       <step>
+        <para>
+   Examine the common issues such as:
+        </para>
+        <variablelist>
+            <varlistentry>
+             <term>Configuration issues </term>
+             <listitem>
+              <para>
+               Verify that the realms, KDC addresses and other settings are correct in the configuration file.
+              </para>
+                   </listitem>
+            </varlistentry>
+            <varlistentry>
+             <term>Synchronized time</term>
+             <listitem>
+              <para>
+               &krb; is sensitive to time discrepancies so ensure all systems have a synchronized clock.
+              </para>
+
+             </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term>Network issues</term>
+                <listitem>
+                 <para>
+                 Verify that there is stable connection between the client and the KDC.
+                 </para>
+                </listitem>
+               </varlistentry>
+               <varlistentry>
+                <term>DNS issues</term>
+                <listitem>
+                 <para>
+                 Ensure there is a proper DNS resolution for the KDC and other &krb; related services.
+                 </para>
+                </listitem>
+               </varlistentry>
+               <varlistentry>
+                <term>Permission issues</term>
+                <listitem>
+                 <para>
+                Check if the &krb; principal has the right permissions.
+                 </para>
+                </listitem>
+               </varlistentry>
+               <varlistentry>
+                <term>Permission issues</term>
+                <listitem>
+                 <para>
+                Once you make the changes, repeat the action that triggered the issue to ensure the problem is solved.
+                Check the <envar>KRB5_TRACE</envar> log to confirm that the issues no longer exists.
+                 </para>
+                </listitem>
+               </varlistentry>
+               <varlistentry>
+                <term>Permission issues</term>
+                <listitem>
+                 <para>
+                Once the issue is resolved, you can disable <envar>KRB5_TRACE</envar> log:
+                 </para>
+                 <screen>&prompt.sudo; unset KRB5_TRACE</screen>
+                </listitem>
+               </varlistentry>
+                </variablelist>
+       </step>
+   </procedure>
+   <note><para>Machine accounts are subject to the same &krb; authentication as user accounts.
+    During &krb; authentication, clients that run local processes using the <literal>system</literal>
+account, assign these processes to the machine account when accessing remote resources. The machine account
+is associated to the computer name registered with the domain controller and is distinct with a <literal>$</literal>
+sign.</para></note>
```

Comment:
This paragraph is very vague and potentially inaccurate. Machine accounts are no different to a user account. They just are representing that that entity is a machine, instead of a human. There isn't some magic that gives a "service" on the machine credentials either, it's done through keytabs that have extracted principal information.

I'm really not sure what you are trying to communicate in this note.